src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -14,18 +15,20 @@ import useViewTracker from '#hooks/useViewTracker';
 import useOptimizelyVariation from '#hooks/useOptimizelyVariation';
 import useToggle from '#hooks/useToggle';
 
-const eventTrackingData = {
```

Comment:
instead of making this a function I'd just move it into the component and add the optimizely object to this object

Reply:
Yeah makes sense, the only downside to this is a duplicated eventTrackingData object within each component [here](https://github.com/bbc/simorgh/blob/cb72a981f19555772ec8d5c5beb2b0b5029f3f2f/src/app/containers/CpsFeaturesAnalysis/index.jsx#L31) and [here](https://github.com/bbc/simorgh/blob/cb72a981f19555772ec8d5c5beb2b0b5029f3f2f/src/app/containers/CpsFeaturesAnalysis/index.jsx#L87). Although I think the single features and analysis story component would never happen, the code is still there for it.